src/server/app_render.jsx

```diff
@@ -88,20 +74,11 @@ async function appRender(ctx) {
             userPreferences,
             offchain,
         });
-
-        // Assets name are found in `webpack-stats` file
-        const assets_filename =
-            ROOT +
-            (process.env.NODE_ENV === 'production'
-                ? '/tmp/webpack-stats-prod.json'
-                : '/tmp/webpack-stats-dev.json');
-        const assets = require(assets_filename);
-
         // Don't cache assets name on dev
+        // TODO: Investigate removing this.
         if (process.env.NODE_ENV === 'development') {
```

Comment:
what happens when `process.env.NODE_ENV === 'development'`? it seems like the assets data required at startup will be removed but it will not be restored.